source/pdo_sqlsrv/pdo_dbh.cpp

```diff
@@ -38,6 +38,7 @@ namespace PDOConnOptionNames {
 
 const char Server[] = "Server";
 const char APP[] = "APP";
+const char AccessToken[] = "AccessToken";
```

Comment:
Is this an ODBC keyword as well?

Reply:
No, but similar to TraceFile and TraceOn, this one is also a pre-connection attribute.